src/uct/api/uct_def.h

```diff
@@ -710,6 +761,29 @@ typedef ssize_t
                                        *pack_args, void *priv_data);
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Callback to notify that the client side endpoint is bound to device.
+ *
+ * This callback routine will be invoked, when the client side endpoint is bound
+ * to specific network device, that means what address and route to the peer is
```

Comment:
.. bound to a local device. [drop the 2nd part of the sentence. address/route do not reflect in the API]